fe/fe-core/src/main/java/org/apache/doris/statistics/ColStatsData.java

```diff
@@ -185,4 +185,24 @@ public ColumnStatistic toColumnStatistic() {
             return ColumnStatistic.UNKNOWN;
         }
     }
+
+    public boolean isNull(String value) {
+        return value == null || value.equalsIgnoreCase("NULL");
```

Comment:
why need `value.equalsIgnoreCase("NULL")` ?

Reply:
Will fix it in a separate pr.